bc_obps/service/operation_service_v2.py

```diff
@@ -159,3 +167,126 @@ def register_operation_operation_representative(
         operation.contacts.set(contact_ids_to_add_to_operation)
         operation.set_create_or_update(user_guid)
         return operation
+
+    # @classmethod
+    # @transaction.atomic()
+    # def create_registration_purpose(cls, user_guid: UUID, operation_id: UUID, payload: RegistrationPurposeIn) -> Dict[str, Union[str, UUID]]:
+
+    #     registration_payload = RegistrationPurposeIn(
+    #         registration_purpose=payload.registration_purpose, regulated_products=payload.regulated_products
+    #     )
+    #     cls.set_registration_purpose(user_guid, operation_id, registration_payload)
+
+    @classmethod
+    @transaction.atomic()
+    def create_or_update_operation_v2(
+        # brianna better to do as a default arg?
```

Comment:
thoughts?

Reply:
I don't understand the question...

Reply:
These are the args:
```
cls,
        user_guid: UUID,
        operation_id: UUID | None,
        payload: OperationInformationIn,
```
I've done UUID | None instead of making the default for operation_id None. I did this because optional args have to go at the end and we usually give the ids before the payload, but non-python-y me doesn't know if that's a good decision

Reply:
ah I see. IMO there's nothing wrong with having `operation_id` as the last arg so it can be optional